### PR TITLE
Map Planets things

### PR DESCRIPTION
**Pull request by TheGiraffe3**
Switches the quotation marks to the standard, fixes some typos, and puts tabs in the right places. Also changes a government or two to correspond with the description.

### CODE REVIEW

File: data/map planets.txt [hunk 1]
```diff
@@ -131,71 +131,71 @@ planet Khora-Vasa-Reyyaa
 planet Khosa-Kella-Oa
 	attributes successor uninhabited
 	landscape land/mars1
-	description "Devoid of atmosphere, this small and rocky world sits at the very edge of the galaxy. This may very well be the last planet for millions of light years, the furthest child of the Milky Way."
+	description `Devoid of atmosphere, this small and rocky world sits at the very edge of the galaxy. This may very well be the last planet for millions of light years, the furthest child of the Milky Way.`
 
 planet Kuj-Sol-Nnesa
-	attributes successor uninhabited boldtourism
+	attributes boldtourism successor uninhabite
```
Comment (Daeridanii1) [inline data/map planets.txt]:
```suggestion
	attributes boldtourism successor uninhabited
```

Comment (TheGiraffe3) [reply data/map planets.txt]: Whoops...

**Review by Daeridanii1 — APPROVED** [review]
This all looks good to me. Thank you for your help!

**Comment by TheGiraffe3** [comment]
The reason there is a revert commit is that I clicked the wrong button when trying to delete the branch.

**Pull request merged**